### PR TITLE
docs: tutorial

### LINKED ISSUE #3123: Lack of a page of "relation literal" on the book
Something like:

```elm
let a = from [
      {a = 1, b = 10},
      {a = 2, b = 20},
      {a = 3, b = 30},
      {a = 4, b = 40},
      {a = 5, b = 50},
    ]
 
from a
```

### PR DESCRIPTION
**Pull request by aljazerzen**

### CODE REVIEW

File: web/book/src/tutorial/examples/README.md
Comment (max-sixty) [inline web/book/src/tutorial/examples/README.md:1]: FWIW I think a lot of the examples aren't great, are very old, and we might be better off removing...

**Comment by richb-hanover** [comment]
@aljazerzen @max-sixty What's the fate of this Tutorial section of the documentation? The Relations/Filtering/Aggregation pages are in pretty good shape (if I do say so myself :-) 

How might they to be incorporated? Thanks

**Comment by aljazerzen** [comment]
This PR contains two types of stuff:
- newly written tutorial that is is pretty good shape,
- old material that the tutorial is based on, half of which should be adapted into the new tutorial sections,
- old material that is outdated and should be removed completely.

I'm not sure exactly which file is of which type, so I just left them all in this PR after docs reorganization.

But the finished part could be merged now.

**Comment by richb-hanover** [comment]
Thanks for the update

> But the finished part could be merged now.

I have continued to evolve my branch, and am a few days away from something that would be worth discussing. I'll push when it's ready.

**Comment by max-sixty** [comment]
Awesome! Thank you very much @aljazerzen !

**Comment by max-sixty** [comment]
(FYI am fixing Windows tests, file name issues...)

**Comment by richb-hanover** [comment]
@aljazerzen @max-sixty  I agree these updates are very good. It's helpful to clarify the naming/terminology that we want to use throughout our documentation.

In parallel, I had started to make tweaks to the Tutorial section. I will let the changes settle out (changes for Windows file names? Anything else?) and then look through my notes to see which ones are still relevant.  Thanks again.

**Comment by max-sixty** [comment]
> In parallel, I had started to make tweaks to the Tutorial section. I will let the changes settle out (changes for Windows file names? Anything else?)

Great, yes I expect refinements (and I'm guessing @aljazerzen will be happy to have them too 😄)

I'm fixing windows file names — feel free to send anything in you have! The previous one was good!

**Comment by richb-hanover** [comment]
> (FYI am fixing Windows tests, file name issues...)

@max-sixty Is this related? https://github.com/PRQL/prql/actions/runs/5975547245/job/16211733758?pr=3411

**Comment by richb-hanover** [comment]
PS See #3411

**Comment by eitsupi** [comment]
This and #3411 were published to the web.
Looks good! Thanks.

image

**Pull request merged**